src/store/Search.js

```diff
@@ -101,8 +101,8 @@ export default {
   },
   mutations: {
     // general settings
-    UPDATE_SEARCH_QUERY_FILTERS(state, filters) {
-      state.search.updateFilters(filters);
+    UPDATE_SEARCH_QUERY_FILTERS(state, { filters, merge = true }) {
+      state.search.updateFilters(filters, !merge);
```

Comment:
why the default is merge true and the udateFilters take the opposite?

Reply:
Because `updateFilters` are called in other places. If this does not look nice we can declare this function as `updateFilters(filters, merge = true)` and do the same with `addFilter`.